### ISSUE TITLE
Autoload the javascript framework

### ISSUE THREAD

**Issue by Abhoryo** [Abhoryo/APYJsFormValidationBundle#19]
Autoload the javascript framework if it's not load previously.

**Comment by recipe** [comment]
I disagree. Good practic is to put all javascript code at the bottom of the whole page.

**Comment by Abhoryo** [comment]
I agree with you. We can autoload the script in the bottom of the whole page too.
